pubsub/pubsub_test.go

```diff
@@ -231,37 +225,43 @@ func BenchmarkPubSub(b *testing.B) {
 
 func BenchmarkPubSubMultiTopic(b *testing.B) {
 	const topicCount = 10
-	ctx := context.Background()
+	ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
+	defer cancel()
 	for _, typ := range types {
 		b.Run(typ, func(b *testing.B) {
 			ps := newPubSub(typ)
-			topics := make([]pubsub.Topic, topicCount)
+			topics := make([]string, topicCount)
 			for i := 0; i < topicCount; i++ {
-				topics[i] = ps.NewTopic(ctx, fmt.Sprintf("topic%d", i))
+				topics[i] = fmt.Sprintf("topic%d", i)
 			}
-			time.Sleep(100 * time.Millisecond)
+			mut := sync.RWMutex{}
+			count := 0
+			counts := make([]int, topicCount)
```

Comment:
I can't seem to find there `counts` are being used

Reply:
This was a vestige of previous tests, but we don't need it any more. Thanks.